Makefile

```diff
@@ -14,4 +14,8 @@ lint: $(GOPATH)/bin/golangci-lint
 
 .PHONY: clean
 clean:
-	-rm perfman
+	-rm -f perfman
+
+.PHONY: clean-output
+clean-output:
+	-rm -rf output
```

Comment:
`clean` should clean everything including the output. Are we not putting the output file under perfman? If not, we should.